src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,21 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE = 35;
+    private static final int REQUIRED_YEARS = 10;
+    private static final String REQUIRED_CITIZENSHIP = "Ukrainian";
+
+    @Override
+    public boolean test(Candidate c) {
```

Comment:
NEVER use just letter for naming variables, please
```suggestion
    public boolean test(Candidate candidate) {
```